static/app/components/events/interfaces/spans/types.tsx

```diff
@@ -128,11 +128,6 @@ export type EnhancedProcessedSpanType =
       type: 'span_group_siblings';
     } & SpanSiblingGroupProps);
 
-export type SpanEntry = {
```

Comment:
This was duplicated type and I moved it to the event.tsx where other entries are defined